openepi_client/geocoding/_geocoding_types.py

```diff
@@ -0,0 +1,129 @@
+# generated by datamodel-codegen:
+#   filename:  https://api-test.openepi.io/geocoding/openapi.json
+#   timestamp: 2023-11-14T13:47:49+00:00
+
+from __future__ import annotations
+
+from typing import Any, List, Optional, Union
+
+from pydantic import BaseModel, ConfigDict, Field, RootModel
+from typing_extensions import Literal
+
+
+class Coordinate(RootModel[List[Any]]):
+    root: List[Any]
+
+
+class LineString(BaseModel):
```

Comment:
A bit unfortunate that we can't reuse GeoJSON types, but it's not so bad since they are autogenerated